spec/triggers/aws-cloudwatch.md

```diff
@@ -0,0 +1,137 @@
+# AWS Cloudwatch Trigger
+
+This specification describes the `aws-cloudwatch` trigger for AWS Cloudwatch.
+
+```yaml
+triggers:
+  - type: aws-cloudwatch
+    metadata:
+      # Required: namespace
+      namespace: AWS/SQS
+      # Required: Dimension Name
+      dimensionName: QueueName
+      dimensionValue: keda
+      metricName: ApproximateNumberOfMessagesVisible
+      targetMetricValue: "2"
+      minMetricValue: "0"
+      # Required: region
+      awsRegion: "eu-west-1"    
+```
+
+The `namespace` value is the service you want to get metrics for, in this case 'AWS/SQS'. Many other options exist.
```

Comment:
I like this a lot more, thanks! Maybe a list is nicer to read?

What I'm personally missing is the auth parameters: If I'm new, what options of configuring it in the trigger auth do I have?

This is the spec so you should define, use `awsRoleArn` (what is `arn` btw)? or use `awsAccessKeyID` & `awsSecretAccessKey`.

That should help folks use this trigger.

Reply:
I added a bit more information about the authentication bits.